rfcs/0013-synthea-future-claims-and-automation.md

```diff
@@ -0,0 +1,148 @@
+# RFC Proposal
+[RFC Proposal]: #rfc-proposal
+
+* RFC Proposal ID: `0013-Synthea-future-claims-and-automation`
+* Start Date: 2022-04-08
+* RFC PR: [beneficiary-fhir-data/rfcs#0013]()
+* JIRA Ticket(s):
+  * [BFD-1616](https://jira.cms.gov/browse/BFD-1616)
+
+The demand for realistic synthetic beneficiaries with recent claim data, for BFD users, continues to grow after each Synthea release. While the process to generate, load, test, and release synthetic data has become more refined and convenient, it is prone to human error that can cause significant delays and miscommunication during release. This RFC proposes a streamlined, automated process for generating, loading, testing, and releasing Synthea data to BFD users, while taking into account changes to synthetic benficiary properties with recent and future claim dates. 
+
+## Status
+[Status]: #status
+
+* Status: Proposed
+* Implementation JIRA Ticket(s):
+  * [BFD-XXXX](https://jira.cms.gov/browse/BFD-XXXX)
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+* [RFC Proposal](#rfc-proposal)
+* [Status](#status)
+* [Table of Contents](#table-of-contents)
+* [Motivation](#motivation)
+* [Proposed Solution](#proposed-solution)
+    * [Proposed Solution: Detailed Design](#proposed-solution-detailed-design)
+    * [Proposed Solution: Unresolved Questions](#proposed-solution-unresolved-questions)
+    * [Proposed Solution: Drawbacks](#proposed-solution-drawbacks)
+    * [Proposed Solution: Notable Alternatives](#proposed-solution-notable-alternatives)
+* [Prior Art](#prior-art)
+* [Future Possibilities](#future-possibilities)
+* [Addendums](#addendums)
+
+## Motivation
+[Motivation]: #motivation
+
+For those unfamiliar with Mitre's Synthea application for generating synthetic beneficiary and claim data, before discussing motivation and a proposed solution, here is some background on Synthea and how BFD uses it. 
+
+### Background on Synthea
+
+Mitre Synthea:
+
+* [Synthea](https://github.com/synthetichealth/synthea) -- for generating synthetic beneficiary and claim data to load into BFD DB.
+  * How To Differentiate Synthea And Real Data:
+    - Synthetic beneficiary IDs are a negative number.
+    - Synthetic MBIs have the character 'S' in a certain position.
+
+Generating & Loading Synthea Data: 
+- BFD clones the master branch of the Synthea codebase, and generates the RIF files via the `run_Synthea` command, where the batch size and geography can be set, as well as a Synthea.properties file, which is important, for defining numeric ranges of key claim and beneficiary properties i.e. beneficiary ID, claim group ID, claim date, etc.
```

Comment:
This is true. This section is more a broad overview on how we have generated synthea data in the past, as opposed to a proposed procedure. I can just change this to "download" instead of "clone", so it doesnt imply using git.